### DISCUSSION TITLE
Can I take pictures with another app within QField?

### DISCUSSION THREAD

**Discussion by SophieMBHH** [opengisch/QField#5634] [Q&A]
Hi there,
is there a way to take pictures in QField not using the regular device's camera? Instead my colleagues want to use the app "TimeStamp" which imprints e.g. the location, date and time on the picture. While I know that this is not necessary it would just be more convenient to them. 

so is there a way that when you take a picture within QField to chose another 'camera'?

Thank you in advance!

**Comment by nirvn** [comment]
@SophieMBHH , you'll be happy to learn that QField 3.4.0 has gained capability to imprint such information on photos taken by QField's own camera. 

It'll roll out onto the play store in the coming days. In the meantime you can install it manually from the release page APKs:

https://github.com/opengisch/QField/releases/tag/v3.4.0
